src/webhooks/interfaces/webhook-directory-user.interface.ts

```diff
@@ -1,6 +0,0 @@
-import { DirectoryUser } from '../../directory-sync/interfaces';
-
-export interface WebhookDirectoryUser extends DirectoryUser {
```

Comment:
This type is redundant now since the timestamps were previously added to the base `DirectoryUser`